### PR TITLE
Doc: Adds link to demo; clarifies Rhino

### PR DESCRIPTION
**Pull request by vibalre**
## Changes description

* Adds demo link in README
* Adds note on properly namespacing element when using with Rhino and Shiny modules

### CODE REVIEW

File: vignettes/use-undraw-drawkit-image.Rmd [hunk 1]
```diff
@@ -34,14 +34,17 @@ empty_state <- empty_state_component(
...
 If the image is in `app/static/img/image.png`, you can use the same code as above.
 Update the `src` path accordingly.
 
 ```r
+# In your module's server function
+ns <- session$ns
+
 empty_state_manager <- EmptyStateManager$new(
-    id = "myElement",
+    id = ns("myElement"),
```
Comment (vibalre) [inline vignettes/use-undraw-drawkit-image.Rmd:47]: Just a quick reminder for users to properly namespace elements when creating an empty state manager.

File: README.md [hunk 2]
```diff
@@ -71,6 +71,7 @@ server <- function(input, output, session) {
 shinyApp(ui, server)
 ```
 
+For more examples of empty state components, please see this [demo](https://connect.appsilon.com/content/79ddb63f-5a96-4da3-9d85-0d66c1507737/).
```
Comment (jakubnowicki) [inline README.md:74]: Please change the application link to a custom one and update it here.

**Review by jakubnowicki — APPROVED** [review]
LGTM - please resolve my comment and you can merge.

File: README.md (hunk 2 above)
Comment (vibalre) [reply README.md]: Link should be updated now.

**Pull request merged**